src/toolkits/style_transfer/style_transfer_model_definition.cpp

```diff
@@ -46,7 +58,7 @@ void define_resnet(model_spec& nn_spec, size_t num_styles) {
       /* stride_height */ 1,
       /* stride_width */ 1,
       /* padding */ padding_type::VALID,
-      /* weight_init_fn */ zero_weight_initializer());
+      /* weight_init_fn */ initializer);
```

Comment:
Doesn't this mean we're performing a bunch of random initialization even when we're about to overwrite with the pre-trained model?

Reply:
Yeah, I noticed that as well. I wasn't sure if it was worth it to write in a flag to keep the model uninitialized. It seemed like a small one time startup cost for the model, but I'll add a flag [Boolean] (if you think that's the right way to do it). Or should it be an enum type of initialization for future proofing? What's your suggestion?

Reply:
I think I got it.